resources/public/api/conf/5.0/foreign_property_cumulative_summary_retrieve.yaml

```diff
@@ -0,0 +1,99 @@
+description: |
+  This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.
+
+  Note: This endpoint only supports submissions for tax year 2025-26 onwards.
+
+  ### Test data
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                 |
+  |----------------------------------|--------------------------------------------------------------------------|
```

Comment:
Can you please align the retrieve and the create and amend endpoints? Currently, there is a `Tax Years` column in the create and amend endpoint, which is reflected in both specifications. However, it seems this column hasn't been added to other endpoints, for example, `Create and Amend a Foreign Property Annual Submission`. To maintain consistency, the test data for the create and amend endpoint should be updated to remove this `Tax Years` column. Thanks